cmd/cmd.go

```diff
@@ -1274,7 +1273,6 @@ func NewCLI() *cobra.Command {
 
 	runnerCmd := &cobra.Command{
 		Use:    "runner",
-		Short:  llama.PrintSystemInfo(),
```

Comment:
How was this help output getting used? Doesn't seem directly related to consolidating the logging. I actually can't get it to output at all in any case directly from the runner CLI.

Reply:
This is being removed for a few reasons.

First, there's a custom help for this subcommand so `runner.Short` is never referenced in `ollama runner -h`.

Second, this subcommand is hidden so `runner` and `runner.Short` never appears in `ollama -h`

Third, and most importantly, simplify calling `ollama runner` is no sufficient to get the system information. The scheduler sets up the execution environment which drives 90% of the useful outputs in system information. This includes features such as AVX, Metal, CUDA, ROCm etc.

Reply:
Thanks for the context!